react/src/__tests__/__snapshots__/App.test.js.snap

```diff
@@ -47,9 +47,11 @@ exports[`renders the concepts 1`] = `
       <a
         href="https://virtuoso.hrcpt.online"
         onClick={[Function]}
+        onMouseEnter={[Function]}
+        onMouseLeave={[Function]}
       >
         <div
-          className="sc-gzVnrw guBrVN"
+          className="sc-gzVnrw cdQPlJ"
```

Comment:
If this is going to change a lot, we should know why. Until that we have that kind of data I'd advocate merging the snapshot change.